functional-tests/devscripts/prepare_env.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+yum install docker
```

Comment:
Does this need to be `yum install -y docker`?

Reply:
Yes, sure, fixed. Thank you.